packages/components/src/DateTimePickers/InputDateTimePicker/date-extraction.js

```diff
@@ -380,8 +380,40 @@ function extractPartsFromTextInput(textInput, options) {
 	};
 }
 
+/**
+ * Extract parts (date, time, date/time, textInput) from a value with
+ * different possible types
+ * @param value {string | Date \ number}
+ * @param options {Object}
+ * @returns
+ *	{{
+ *		date: Date,
+ *		time: { hours: string, minutes: string, seconds: string },
+ *		datetime: Date,
+ *		textInput: string
+ * 	}}
+ */
+function extractParts(value, options) {
```

Comment:
imo, this file would be better named by 'normalize.js'

Reply:
No, because we expose extraction functions. Normalize is quite generic and we usually call normalization something that takes a value, transform it and return this same value, but with a defined format.
Here we extract things : date, hours, and convert it : date and string

Reply:
inflate?

Reply:
This word is not really clear neither. But I'll draw the card "Not in this PR scope" as it's not something I just created and don't want to discuss about this naming because the ideas we have for now is not better than the one in place.
But if we find something better later, an easy PR can be done.